src/stavetext.ts

```diff
@@ -31,36 +52,41 @@ export class StaveText extends StaveModifier {
     };
   }
 
-  getCategory() {
+  getCategory(): string {
     return StaveText.CATEGORY;
   }
-  setStaveText(text) {
+
+  setStaveText(text: string): this {
     this.text = text;
     return this;
   }
-  setShiftX(x) {
+
+  setShiftX(x: number): this {
     this.shift_x = x;
     return this;
   }
-  setShiftY(y) {
+
+  setShiftY(y: number): this {
     this.shift_y = y;
     return this;
   }
 
-  setFont(font) {
+  setFont(font: FontInfo): this {
     Vex.Merge(this.font, font);
```

Comment:
We can use the spread operator here.

Reply:
`FontInfo` has no optional member. Equal is enough.